presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergNativeMetadata.java

```diff
@@ -302,6 +305,16 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
         }
 
         Table icebergTable = transaction.table();
+        ReplaceSortOrder replaceSortOrder = transaction.replaceSortOrder();
+        List<SortField> sortFields = replaceSortOrder.apply().fields().stream().map(SortField::fromIceberg)
+                .collect(toImmutableList());
+        try {
+            replaceSortOrder.commit();
```

Comment:
```suggestion
        SortOrder sortOrder = parseSortFields(schema, getSortOrder(tableMetadata.getProperties()));
        ReplaceSortOrder replaceSortOrder = transaction.replaceSortOrder();
        List<org.apache.iceberg.SortField> icebergSortFields = sortOrder.fields();
        List<SortField> sortFields = icebergSortFields.stream().map(SortField::fromIceberg)
                .collect(toImmutableList());
        try {
            for (org.apache.iceberg.SortField sortField : icebergSortFields) {
                if (sortField.direction() == SortDirection.ASC) {
                    replaceSortOrder.asc(schema.findColumnName(sortField.sourceId()), sortField.nullOrder());
                }
                else {
                    replaceSortOrder.desc(schema.findColumnName(sortField.sourceId()), sortField.nullOrder());
                }
            }
```

I think a feasible way could go like this, although not carefully check whether it's the best. The core point is, we should get `sortOrder` from `tableMetadata` and then set it into the create table transaction through `replaceSortOrder`. You can have a try.

Reply:
@hantangwangd Done the changes.